SmartTutor/Startup.cs

```diff
@@ -81,7 +82,9 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<IWorkspaceCreator, NoWorkspaceCreator>();
             services.AddScoped<ILearnerRepository, LearnerDatabaseRepository>();
 
-            services.AddScoped<IInstructor, VARKRecommender>();
+            services.AddScoped<IPrerequisiteSelectionStrategy, RandomizedPrerequisiteSelectionStrategy>();
+            services.AddScoped<IInstructor, VARKInstructor>();
```

Comment:
Could you explain how lines 86 and 87 work together?

This is something we should have in the BSc as well, to explain the whole configuration setup.

Reply:
Yes, I've read the article and am going to include it in my thesis :smile: